### ISSUE TITLE
Is there a docset for the API somewhere?

### ISSUE THREAD

**Issue by danieltorrecillas** [xmartlabs/Eureka#1710]
I use Dash.app to search and read documentation. One source you can download docsets from is http://cocoadocs.org . When searching for "Eureka" on cocoadocs.org, there is a result, but the site doesn't actually show any API documentation. It just shows the `README.md` from the repository. It says `Error Parsing Pod [...] Could not find Objective-C Classes.`

http://cocoadocs.org/docsets/Eureka/4.0.1/

Is there an alternate site where a docset is located?

**Comment by mats-claassen** [comment]
No, there is no alternate site. I haven't checked CocoaDocs before, but it is weird that it shows `4.0.1` as the latest version while 4.3.0 is the latest one on CocoaPods.

From what I saw on the web the error you posted is related to CocoaDocs trying to parse the library with the wrong Swift version. It will not find any Objective-C classes here.

**Comment by danieltorrecillas** [comment]
> it is weird that it shows 4.0.1 as the latest version while 4.3.0 is the latest one on CocoaPods.

@mats-claassen, do you know how to correct this? If you could point me in the right direction, I'd be happy to do as much work as allowed.

**Comment by mats-claassen** [comment]
No I don't. I haven't used CocoaDocs. It is possible that they need to run some kind of update

**Comment by danieltorrecillas** [comment]
@orta, is there an alternative or solution you could see for this repository?

**Comment by orta** [comment]
Nope, no-one's taken that mantle I'm afraid
